jsondb.js

```diff
@@ -116,6 +116,60 @@ export const tryPost = ({ ...props }) => {
   return { status: 400 };
 };
 
+export const tryPut = ({ ...props }) => {
+  if (props.method !== "PUT") {
+    return props;
+  }
+  const { lensPath, json, body } = props;
+  const path = [...lensPath];
+  const view = U.view(U.lensPath(path))(json);
+  const parentPath = path.slice(0, -1);
+  const parentView = U.view(U.lensPath(parentPath))(json);
+  if (!view && parentView) {
```

Comment:
Again, can we have here descriptive condition like `const whatIsThisFor = !view && parentView`?